_includes/default.njk

```diff
@@ -78,17 +78,14 @@ site:
         <h1>{{wnp.type}}: {{wnp.description}}</h1>
         <p>
           This is a Work-Needing and Prospective Page.
+          {% if wnp.type == "O" %}
+          Orphaned: no one is maintaining this page.
+          {% elif wnp.type == "RFA" %}
```

Comment:
Should display: `<strong>R</strong>equest <strong>F</strong>or <strong>A</strong>doption: this page is maintained but up for adoption.`

Reply:
And for type `RFH`, should display `<strong>R</strong>equest</strong> <strong>F</strong>or <strong>H</strong>elp: this page is maintained but needs help.